package.json

```diff
@@ -40,6 +40,7 @@
     "ethjs-abi": "^0.2.1",
     "lodash.flatten": "^4.4.0",
     "semver": "^5.6.0",
+    "truffle-contract": "^4.0.31",
```

Comment:
What do you think about making this an implicit dependency? I.e. declare as a `devDependency` to make sure we have it for development and testing, but assume that it will be available for the user when we detect a Truffle environment.

It feels better to not force people to install a dependency if they won't need it. We may run into issues with mismatching versions though. Actually, that's an issue either way because otherwise we could be returning an instance of `truffle-contract` of a version other than the one being used by the user.

Reply:
Hm, that's a good idea. We may want to also add a custom error message in that case.